src/index.js

```diff
@@ -7,22 +7,37 @@ const mealsRow = document.querySelector('.meals-row');
 // get all categories to add event listeners
 const categories = document.querySelectorAll('.category');
 
+const showLikes = async (id) => {
+  let likes = 0;
+  const mealLikes = await Fetch.getLikes();
+  await mealLikes.forEach((meal) => {
```

Comment:
- [ ] [OPTIONAL]  You can make use or` filter()` instead of f`orEach()` . With` filter()` this block can be compiled in just one line